source/user-manual/reference/ossec-conf/wodle-s3.rst

```diff
@@ -401,6 +392,19 @@ Usage example:
 
     <discard_regex field="data.configurationItemStatus">REJECT</discard_regex>
 
+.. _bucket_remove_from_bucket:
+
+remove_from_bucket
+~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
The new `remove_from_bucket` title should be an H3, not an H2. 

![image](https://user-images.githubusercontent.com/61882981/216043915-ee4f0598-33c5-4c3e-9576-5377a0dd9662.png)